src/net_processing.cpp

```diff
@@ -58,9 +58,9 @@ static constexpr int32_t MAX_OUTBOUND_PEERS_TO_PROTECT_FROM_DISCONNECT = 4;
 /** Timeout for (unprotected) outbound peers to sync to our chainwork, in seconds */
 static constexpr int64_t CHAIN_SYNC_TIMEOUT = 20 * 60; // 20 minutes
 /** How frequently to check for stale tips, in seconds */
```

Comment:
```suggestion
/** How frequently to check for stale tips */
```

Reply:
Addressed in 1a1fafa4fdcbe7b0d1ef8df466a71bd3e65195d3